packages/manager/modules/pci/src/projects/project/public-gateways/onboarding/onboarding.routing.js

```diff
@@ -0,0 +1,27 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('pci.projects.project.public-gateways.onboarding', {
+    url: '/onboarding',
+    component: 'pciProjectPublicGatewaysOnboarding',
+    redirectTo: (transition) =>
+      transition
+        .injector()
+        .getAsync('privateNetworks')
+        .then((privateNetworks) =>
+          privateNetworks.length < 0
```

Comment:
Here, I need to check `privateNetworks.length > 0`. I will take care of this in next commit where I have to change illustration image and some more condition for `create a public gateway` button. Waiting for some of the queries to get answered.